corelib-record/src/main/java/eu/europeana/corelib/record/RecordService.java

```diff
@@ -0,0 +1,82 @@
+package eu.europeana.corelib.record;
+
+import eu.europeana.corelib.definitions.edm.beans.FullBean;
+import eu.europeana.corelib.edm.exceptions.BadDataException;
+import eu.europeana.corelib.web.exception.EuropeanaException;
+
+/**
+ * Retrieves CHO records from Mongo database.
+ * If the provided recordId is not known, the redirect database can be checked to see if this recordId was known in the
+ * past and has gotten a new id
+ * @see FullBean
+ *
+ * @author Patrick Ehlert
+ * Created on 4-4-2020
+ */
```

Comment:
can't we get rid of the interface - impl pattern right away for cases like this?